textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -10,6 +12,16 @@
 
 public class RegexToGramQueryTranslatorTest {
 	
+	private void printTranslatorResult(String regex) {
+		GramBooleanQuery exactQuery = RegexToGramQueryTranslator.translate(regex);
+		
+		System.out.println("regex: "+regex);
```

Comment:
We cannot automate the testing process using `println`.  Since we have a function to test the equivalence of two boolean expressions, can we use it in the test it to validate the logic?